activiti-cloud-starter-configure/src/main/java/org/activiti/spring/bpmn/parser/CloudActivityBehaviorFactory.java

```diff
@@ -0,0 +1,23 @@
+package org.activiti.spring.bpmn.parser;
+
+import org.activiti.bpmn.model.Signal;
+import org.activiti.bpmn.model.SignalEventDefinition;
+import org.activiti.bpmn.model.ThrowEvent;
+import org.activiti.engine.impl.bpmn.behavior.IntermediateThrowSignalEventActivityBehavior;
+import org.activiti.engine.impl.bpmn.parser.factory.DefaultActivityBehaviorFactory;
+import org.activiti.engine.impl.context.Context;
+import org.activiti.spring.SpringProcessEngineConfiguration;
+
+
+public class CloudActivityBehaviorFactory extends DefaultActivityBehaviorFactory {
+
+    public static final String DEFAULT_THROW_SIGNAL_EVENT_BEAN_NAME = "defaultThrowSignalEventBehavior";
+
+    @Override
+    public IntermediateThrowSignalEventActivityBehavior createIntermediateThrowSignalEventActivityBehavior(ThrowEvent throwEvent,
```

Comment:
@daisuke-yoshimoto it will be great if we can avoid having this class and just extending the engine in the same way as ServiceTasks.. which is using the expressionManager to choose which implementation to use.

Reply:
@Salaboy 
Expression is a mechanism for calling a Spring bean from a service task in BPMN.
https://spring.io/blog/2016/02/07/orchestrating-work-with-activiti-and-spring-integration

If it is a statically determined bean like this time, we can get the bean from the ApplicationContext normally without using Expression.

Reply:
@daisuke-yoshimoto I was wondering if we can do the same for Signal Behaviour resolution. So we don't need to do an extra configuration on the Cloud Environment, it should just work based on what is available.